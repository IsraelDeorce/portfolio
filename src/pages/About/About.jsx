function About() {
  return (
    <>
      <h1>This is the About page!</h1>
      <p>More information are soon to come! ;)</p>
    </>
  )
}

export default About;