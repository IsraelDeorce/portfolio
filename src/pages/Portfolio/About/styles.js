import styled from 'styled-components'

export const TextAbout = styled.div`
  display: flex;
  flex-direction: column;
  @media (min-width: 992px) {
    padding: 0 10%;
  }
`
