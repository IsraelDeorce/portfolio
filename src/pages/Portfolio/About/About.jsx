import { useTranslation } from 'react-i18next'
import * as S from './styles'

function About() {
  const { t } = useTranslation()
  return (
    <section id='about'>
      <h2>{t('button.about_me')}</h2>
      <S.TextAbout>
        <p>{t('desc.about_1')}</p>
        <p>{t('desc.about_2')}</p>
      </S.TextAbout>
    </section>
  )
}

export default About
