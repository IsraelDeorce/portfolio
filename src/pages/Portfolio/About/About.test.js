import About from './About'

describe('<About/>', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Snapshot', () => {
    it('renders without crashing using shallow', () => {
      expect(shallow(<About />)).toMatchSnapshot()
    })
  })
})
