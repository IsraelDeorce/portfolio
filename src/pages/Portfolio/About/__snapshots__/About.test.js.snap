// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About/> Snapshot renders without crashing using mount 1`] = `
<About>
  <section
    id="about"
  >
    <h2>
      button.about_me
    </h2>
    <styled.div>
      <div
        className="sc-bdnxRM cYgjXS"
      >
        <p>
          desc.about_1
        </p>
        <p>
          desc.about_2
        </p>
      </div>
    </styled.div>
  </section>
</About>
`;

exports[`<About/> Snapshot renders without crashing using render 1`] = `
<section
  id="about"
>
  <h2>
    button.about_me
  </h2>
  <div
    class="sc-bdnxRM cYgjXS"
  >
    <p>
      desc.about_1
    </p>
    <p>
      desc.about_2
    </p>
  </div>
</section>
`;

exports[`<About/> Snapshot renders without crashing using shallow 1`] = `
<section
  id="about"
>
  <h2>
    button.about_me
  </h2>
  <styled.div>
    <p>
      desc.about_1
    </p>
    <p>
      desc.about_2
    </p>
  </styled.div>
</section>
`;
